docker/docker-compose-local.yml

```diff
@@ -30,7 +30,7 @@ services:
         volumes:
             - ${HDW_AIRFLOW_ROOT}/dags:/usr/local/airflow/dags
             - ${HDW_AIRFLOW_ROOT}/plugins:/usr/local/airflow/plugins
-            - ${HDW_AIRFLOW_ROOT}:/usr/local/airflow/requirements
+            - ${HDW_AIRFLOW_ROOT}/requirements:/usr/local/airflow/requirements
```

Comment:
I think it should be `${HDW_AIRFLOW_ROOT}/requirements.txt:/usr/local/airflow/requirements/requirements.txt` 
since we have `requirements.txt` under folder harrys_airflow/ right?

Reply:
Moved this in the HDW PR for aftership into it's own directory. Mounting the file itself doesn't seem to play nice with docker.